instances/main.tf

```diff
@@ -18,7 +18,7 @@ module "x64-large-build-pool" {
   azp_token            = var.azp_token
   disk_size_gb         = 2000
   idle_instances_count = 1
-  instance_type        = "r5.8xlarge"
+  instance_types       = ["r5a.8xlarge", "r5.8xlarge"]
```

Comment:
Are we sure r5a's give us the same performance, they have a significantly less powerful cpu, are we sure we don't need to go up to: `r5a.12xlarge` to average around the same times?

Reply:
The AMD EPYC processor itself is not significantly less performant, though it seems it has less I/O performance though that's not where we are bound. I didn't see any obvious performance drop so this is OK.

Reply:
Okay just wanted to make sure we've at least tested this.